app/controllers/weets.controller.js

```diff
@@ -0,0 +1,25 @@
+const { weetCreate } = require('../services/weets');
+const { weetSerializer } = require('../serializer/weets');
+const { weetCreateMapper } = require('../mappers/weets');
+const { getSentence } = require('../services/numbers');
+const { maxLength } = require('../helpers/validation');
+const { DEFAULT_ERROR } = require('../constants/errors');
+const {
+  common: { weets }
+} = require('../../config');
+const errors = require('../errors');
+const logger = require('../logger');
+
+exports.weetCreate = async (req, res, next) => {
+  const { id } = req.payload;
+  try {
+    const weetContent = await getSentence();
+    if (!maxLength(weetContent, weets.contentMaxLength)) throw errors.defaultError(DEFAULT_ERROR);
```

Comment:
I don't think this is a good approach to handle the limitation. This is not a user (consumer) error.
I think you could request a new sentence or crop the actual one.

Reply:
Perfect. I decided crop it.